single_tech_samples/synapseanalytics/synapseserverless/README.md

```diff
@@ -0,0 +1,179 @@
+- Wait for the container to build, check the logs for more information:
+  
+  ![Container Logs](./images/container.jpg)
+
+### Without using Dev Containers
+
+Clone the repository and follow this prerequisites in order to run the sample:
+
+- Login to Azure az login
+- Install jq
+    ```sudo apt-get install jq```
+- Install Az CLI
+    ```curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash``` (for Ubunto or Debian).
+- Define the following environment variables:
+  - export PROJECT=<PROJECT_NAME>
+  - export AZURE_SUBSCRIPTION_ID=<SUBSCRIPTION_ID>
+  - export DEPLOYMENT_ID=dep1 (or any other string). NOTE: if the deployment fails, you need either to delete the resources created or use a different DEPLOYMENT_ID.
+  - export AZURE_LOCATION=westeurope (or any other Azure location)
+
+NOTE: if you need to check the list of env variables use ```printenv```. If you need to delete an env variable use ```unset <NAME_USED_ON_THE_EXPORT>```
+
+## Running the Sample
+
+Navigate to the directory /modern-data-warehouse-dataops/single_tech_samples/synapseanalytics/synapseserverless and run the following command:
+
+```bash -c ./deploy.sh```
+
+Once the sample is deployed in the subscription of choice, navigate to Synapse Studio and either wait for the trigger to start the Pl_NYCTaxi_0_Main pipeline or you can trigger it manually. This pipeline is reponsible to call P_NYCTaxi_1_Setup and P_NYCTaxi_2_Preparation pipelines is sequence.
+
+Once the pipeline finishes successfully, you can navigate through the db_serveless database and check the views that were created and experiment the CLS scripts as explained later in this document.
+
+## Infrastructure deployment
+
+The script firstly deploys the infrastructure that is required to demonstrate the objectives of the sample.
+The infrastructure is composed by:
+
+- one resource group, named <PROJECT_NAME><DEPLOYMENT_ID>-rg, and within the resource group you can find:
+  - one storage account for internal Synapse use, named <PROJECT_NAME>st2<DEPLOYMENT_ID>
+  - one storage account that acts as the data lake to store the NYC Taxi Data, named <PROJECT_NAME>st1<DEPLOYMENT_ID>
+  - a Synapse workspace that includes a Serverless pool and a Spark pool, named <PROJECT_NAME>st1<DEPLOYMENT_ID>
+  - one AAD group, named AADGR<PROJECT_NAME><DEPLOYMENT_ID>
+- one resource group named <PROJECT_NAME>-syn-mrg-<DEPLOYMENT_ID>, for  internal use of Synapse
+
+## Synapse Artifacts
+
+The synapse artifacts are uploaded immediately after the infrastructure is successfully deployed and below you can find the list of what should be deployed in the Synapse workspace:
+
+- Datasets
+  - Ds_NYCTaxi_ADLS2
+  - Ds_NYCTaxi_HTTP
+- Linked Services
+  - Ls_NYCTaxi_ADLS2
+  - Ls_NYCTaxi_HTTP
+  - Ls_NYCTaxi_Synapse_Serverless_master
+  - Ls_NYCTaxi_Synapse_Serverless_db
+- Pipelines
+  - Pl_NYCTaxi_0_Main
+  - Pl_NYCTaxi_1_Setup
+  - Pl_NYCTaxi_2_Preparation
```

Comment:
That was the original name :), then I went for something more generic because that pipeline creates the serverless views but also ingests the data - so then I've called it preparation. But I think your comment is good. I will revert to the original name so it brings more clarity.